### PR TITLE
updated readme with citation and new parser creation

### PR DESCRIPTION
**Pull request by botondbarta**

### CODE REVIEW

**Review by attilanagy234 — APPROVED** [review]
Just two minor comments, otherwise LGTM

**Comment by juditacs** [comment]
Thank you for doing this. Hopefully we will see similar corpora in other languages :heart_eyes:

File: README.md [hunk 1]
```diff
@@ -80,8 +80,48 @@ cd summarization
 python entrypoints/deduplicate.py preprocess.yaml
 ```
 
+## How to add your own parser
+To add a new parser for the corpus creation process, follow these steps:
+#### 1. Create a new parser class for the specific website.
+Place the parser in the html_parsers package. The parser should inherit from the ParserBase class and implement the following methods:
+```python
+class MyNewWebsiteParser(ParserBase):
```
Comment (NyistMilan) [inline README.md:88]: Maybe add the *remove_unnecessary_text_from_article* function to the class.

**Review by NyistMilan — APPROVED** [review]
1 optional comment, otherwise it looks good!

**Pull request merged**